scripted-tests/run/java-io-file-1/tests/CanonicalPathTest.scala

```diff
@@ -1,7 +1,8 @@
 object CanonicalPathTest {
   import Files._
+  import scala.scalanative.runtime.Platform
 
-  def main(args: Array[String]): Unit = {
+  def main(args: Array[String]): Unit = if (!Platform.isWindows) {
```

Comment:
@It looks like most of the File I/O tests are disabled. How do we distinguish what's supported and what's not?